### PR TITLE
{lang}[intel/2015a] Python 2.7.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-2.7.10-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,120 @@
+name = 'Python'
+version = '2.7.10'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/p/Python/Python-2.7.10-intel-2015a.eb]: Line to long, use `"""` and split over 2 lines

File: easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('Tcl', '8.6.4'),
+]
+
+parallel = 1
+
+sanity_check_paths = {
+    'files': ['bin/sqlite3'],
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb]: you probably also want to check for the `.so` file?

File: easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb]: you're also author?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/aab6c9ae7294e0191bbb for a full test report.

File: easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb]: ok, will fix

File: easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb (hunk 3 above)
Comment (boegel) [inline easybuild/easyconfigs/s/SQLite/SQLite-3.8.10.2-intel-2015a.eb]: I just bumped the version

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/96d20c0803cf868972ae for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/08e17a43bbfe75a09242 for a full test report.

**Comment by JensTimmerman** [comment]
ok

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/395afc027196aee4935e for a full test report.

**Comment by boegel** [comment]
Thanks for the review @JensTimmerman, and testing @wpoely86!

**Pull request merged**